tests/test_DG_advection.py

```diff
@@ -13,9 +13,7 @@ def setup_DGadvection(dirname, vector=False):
 
     mesh = IcosahedralSphereMesh(radius=R,
                                  refinement_level=refinements)
-    global_normal = Expression(("x[0]/sqrt(x[0]*x[0]+x[1]*x[1]+x[2]*x[2])",
-                                "x[1]/sqrt(x[0]*x[0]+x[1]*x[1]+x[2]*x[2])",
-                                "x[2]/sqrt(x[0]*x[0]+x[1]*x[1]+x[2]*x[2])"))
+    global_normal = Expression(("x[0]", "x[1]", "x[2]"))
```

Comment:
Oh, ok.


________________________________
From: Lawrence Mitchell <notifications@github.com>
Sent: 12 April 2016 18:19
To: firedrakeproject/dcore
Cc: Shipton, Jemma
Subject: Re: [firedrakeproject/dcore] Dg advection (#23)


In tests/test_DG_advection.py<https://github.com/firedrakeproject/dcore/pull/23#discussion_r59415699>:

> @@ -13,9 +13,7 @@ def setup_DGadvection(dirname, vector=False):
>
>      mesh = IcosahedralSphereMesh(radius=R,
>                                   refinement_level=refinements)
> -    global_normal = Expression(("x[0]/sqrt(x[0]*x[0]+x[1]*x[1]+x[2]*x[2])",
> -                                "x[1]/sqrt(x[0]*x[0]+x[1]*x[1]+x[2]*x[2])",
> -                                "x[2]/sqrt(x[0]*x[0]+x[1]*x[1]+x[2]*x[2])"))
> +    global_normal = Expression(("x[0]", "x[1]", "x[2]"))


It's not just when R is one. You never have to provide a normalised vector here. We just use it for the direction.

-
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/firedrakeproject/dcore/pull/23/files/f322e474cebe58e9debfb0d41281be1f458c7392..ae4fb60a28ba03ce743765f30d40851c58ff5b00#r59415699>